antarest/study/storage/abstract_storage_service.py

```diff
@@ -315,14 +314,16 @@ def export_output(self, metadata: T, output_id: str, target: Path) -> Path:
         logger.info(f"Exporting output {output_id} from study {metadata.id}")
 
         path_output = Path(metadata.path) / "output" / output_id
-        if not path_output.exists():
+        path_output_zip = Path(metadata.path) / "output" / f"{output_id}.zip"
+        if not path_output.exists() and not path_output_zip.exists():
             raise StudyOutputNotFoundError()
         stopwatch = StopWatch()
-        filename = shutil.make_archive(
-            base_name=os.path.splitext(target)[0],
-            format="zip",
-            root_dir=path_output,
-        )
+        if not path_output_zip.exists():
```

Comment:
else copy the path_output_zip file to target
because it is where the zipped output is to be found (exported)